pkg/services/tree/signature.go

```diff
@@ -23,9 +26,20 @@ type message interface {
 	SetSignature(*Signature)
 }
 
-// verifyClient verifies that the request for a client operation was either signed by owner
-// or contains a valid bearer token.
-func (s *Service) verifyClient(req message, cid cidSDK.ID, rawBearer []byte, op eacl.Operation) error {
+func aclErr(op acl.Op) error {
+	return fmt.Errorf("access to operation %s is denied by basic ACL check", op)
+}
+
+func eaclErr(op eacl.Operation, err error) error {
+	return fmt.Errorf("access to operation %s is denied by extended ACL check: %w", op, err)
+}
+
+// verifyClient verifies that the request for a client operation was either signed by an
```

Comment:
Not sure this sentence is grammatically correct :eyeglasses:

Reply:
incorrectly changed that comment while was changing the code. fixed, could you check it one more time?